app/indexers/parent_issue_indexer.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+class ParentIssueIndexer
+  delegate :query_service, to: :metadata_adapter
+  attr_reader :resource
+  def initialize(resource:)
+    @resource = resource
+  end
+
+  def to_solr
+    return {} unless resource.is_a?(Coin)
+    return {} unless parents.first
+
+    parent_attributes.map { |k, v| ["#{k}_tesim", v] }.to_h
```

Comment:
We should probably prefix these, otherwise when you load the resource from solr (which admittedly we don't do), it'll have a bunch of the metadata from a coin's issue.

Reply:
Sorry. I merged this without reloading the page. Your comment just came up!

Reply:
That's an important comment. Created an issue for it: https://github.com/pulibrary/figgy/issues/2165